src/bech32.h

```diff
@@ -21,8 +21,8 @@
 namespace bech32
 {
 
-/** The Bech32 and Bech32m checksum size */
```

Comment:
In c1a5d5c100b1628456acfa6129e303737f0ad4d3: Any specific reason to remove the comment?

Reply:
it's in `bech32` and is called `CHECKSUM_SIZE` - what does the comment add to that?

Reply:
Maybe that this value works for both bech32 and bech32m?

Reply:
The exceptional case is when they're different (see https://github.com/bitcoin/bitcoin/pull/30623/files#diff-9607135f3b89a188b92ae678f50685a3f65f4e8ef7832e653cc344ec623100dfR39), we assume otherwise that these are common.